src/redux/ensRegistration.ts

```diff
@@ -432,6 +475,38 @@ export const updateTransactionRegistrationParameters = (
   });
 };
 
+export const removeRegistrationByName = (
+  accountAddress: EthereumAddress,
```

Comment:
seems like the `accountAddress` from `useAccountSettings` is always passed in - any reason this isn't just picked up from `getState` `settings` instead of being passed in as a param?

Reply:
makes sense, no real reason